packages/editor/src/components/provider/use-block-editor-settings.js

```diff
@@ -61,11 +63,51 @@ function useBlockEditorSettings( settings, hasTemplate ) {
 			),
 			hasResolvedLocalSiteData: hasFinishedResolvingSiteData,
 			baseUrl: siteData?.url || '',
+			userCanCreatePages: select( coreStore ).canUser(
+				'create',
+				'pages'
+			),
+			userCanCreatePosts: select( coreStore ).canUser(
+				'create',
+				'posts'
+			),
```

Comment:
We already have the `canUser` selector on line 41; let's reuse it.

Also, if you decide to go with just "Page" creation, we can only check for pages.

Reply:
Good spot. I didn't notice.

Updated and also simplified things to focus on Pages only (for now).